firemonsecuritymanager_connector.py

```diff
@@ -0,0 +1,490 @@
+        action_result.add_data(response)
+
+        # Add a dictionary that is made up of the most important values from data into the summary
+        summary = action_result.update_summary({})
+        summary['message'] = response['message']
+
+        # Return success, no need to set the message, only the status
+        # BaseConnector will create a textual message based off of the summary dictionary
+        return action_result.set_status(phantom.APP_SUCCESS)
+
+    def _handle_get_device_group(self, param):
+        # use self.save_progress(...) to send progress messages back to the platform
+        self.save_progress("In action handler for: {0}".format(self.get_action_identifier()))
+
+        # Add an action result object to self (BaseConnector) to represent the action for this param
+        action_result = self.add_action_result(ActionResult(dict(param)))
+
+        # Access action parameters passed in the 'param' dictionary
+
+        # Required values can be accessed directly
+        firemon_domain_id = param['firemon_domain_id']
+        firemon_device_group_id = param['firemon_device_group_id']
+
+        # make rest call
+        ret_val, response = self._make_rest_call(
+            '/domain/' + str(firemon_domain_id) + '/devicegroup/' + str(firemon_device_group_id),
+            action_result,
+            params=None,
+            headers=None
+        )
+
+        if phantom.is_fail(ret_val):
+            # the call to the 3rd party device or service failed, action result should contain all the error details
+            return action_result.get_status()
+
+        # Now post process the data,  uncomment code as you deem fit
+
+        # Add the response into the data section
+        action_result.add_data(response)
+
+        # Add a dictionary that is made up of the most important values from data into the summary
+        summary = action_result.update_summary({})
+        summary['id'] = response['id']
+        summary['domainId'] = response['domainId']
+        summary['name'] = response['name']
+
+        # Return success, no need to set the message, only the status
+        # BaseConnector will create a textual message based off of the summary dictionary
+        return action_result.set_status(phantom.APP_SUCCESS)
+
+    def _handle_get_domain(self, param):
+        # use self.save_progress(...) to send progress messages back to the platform
+        self.save_progress("In action handler for: {0}".format(self.get_action_identifier()))
+
+        # Add an action result object to self (BaseConnector) to represent the action for this param
+        action_result = self.add_action_result(ActionResult(dict(param)))
+
+        # Access action parameters passed in the 'param' dictionary
+
+        # Required values can be accessed directly
+        firemon_domain_id = param['firemon_domain_id']
+
+        # make rest call
+        ret_val, response = self._make_rest_call(
+            '/domain/' + str(firemon_domain_id),
```

Comment:
Better practice to use string formatters

Reply:
Used string format() function.